app/client/models/AppModel.ts

```diff
@@ -451,13 +451,27 @@ export class AppModelImpl extends Disposable implements AppModel {
     return Computed.create(this, this.themePrefs, prefersDarkModeObs(),
       (_use, themePrefs, prefersDarkMode) => {
         let appearance: ThemeAppearance;
+
+        const urlParams = urlState().state.get().params;
+        if (urlParams?.themeAppearance) {
+          themePrefs.appearance = urlParams?.themeAppearance;
```

Comment:
Assigning to `themePrefs` could be problematic because it mutates it.

What if instead we did something like this?

```javascript
        let {appearance, syncWithOS} = themePrefs;

        const urlParams = urlState().state.get().params;
        if (urlParams?.themeAppearance) {
          appearance = urlParams.themeAppearance;
        }

        if (urlParams?.themeSyncWithOs !== undefined) {
          syncWithOS = urlParams.themeSyncWithOs;
        }
        if (syncWithOS) {
          appearance = prefersDarkMode ? 'dark' : 'light';
        }

        let colors: ThemeName | ThemeColors = themePrefs.colors[appearance];
        if (urlParams?.themeName) {
          colors = urlParams.themeName;
        }
        if (typeof colors === 'string') {
          colors = getThemeColors(colors);
        }

        return {appearance, colors};
```